x-pack/plugins/secops/public/components/page/navigation/breadcrumbs.test.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { mount } from 'enzyme';
+import React from 'react';
+import { MemoryRouter } from 'react-router';
+
+// @ts-ignore Prevent auto-format from deleting - needed for 'chrome/ui' imports in host_details/ip_details
+jest.mock('ui/chrome', () => ({
+  getBasePath: () => '',
+}));
+
+import { encodeIpv6 } from '../../../containers/helpers';
```

Comment:
Optional - Might belong more in `lib` folder. trying to keep container just our liaison between graphql and the components